jb/fftw/ut_time_delay_estimator_many.cpp

```diff
@@ -17,14 +19,15 @@ BOOST_AUTO_TEST_CASE(fftw_time_delay_estimator_many_3_dim_tde_with_0) {
   int const nsamples = 1 << 15;
   int const S = 20;
   int const V = 4;
-  int const argmax_tol = 2;
+  int const argmax_tol = 3;
```

Comment:
ACK. I do not know what you did with the library but now tolerance = 1 tests are passing but for complex<float> which make sense because of the argument of a complex number is yet another computation sqrt(a^2 - b^2) if my memory does not fail here. 
Anyway, tolerances are back to 1 except for the aforementioned combination, where a comment was added.